Plugins/NativeCamera/Source/Android/NativeCameraImpl.cpp

```diff
@@ -274,6 +274,8 @@ namespace Babylon::Plugins
             {
                 throw std::runtime_error{"Unable to restore GL context for camera texture init."};
             }
+
+            return static_cast<const Camera::CameraDimensions*>(&m_cameraDimensions);
```

Comment:
Why is this cast here?

Reply:
If you don't cast it as const then compile tries to create an invalid arcana::expected type. I'm going to change to returning the object by value, so this will go away.